core/src/main/java/org/infinispan/lifecycle/ModuleLifecycle.java

```diff
@@ -22,6 +22,10 @@
  */
 public interface ModuleLifecycle {
 
+    default String name() {
+        return this.getClass().getName();
```

Comment:
I would not bother adding a proper name. Let's just use the full class name in addModuleDependency

Reply:
This was just to avoid having strings with FQCN, less ugly, Now this is already done, let's keep with short names?

Reply:
I will remove the name

Reply:
removed